web/static/js/chat.js

```diff
@@ -0,0 +1,22 @@
+let Chat = exports.Chat = function(inputSelector, outputSelector) {
```

Comment:
No idea if this is the idiomatic way to export a function... I first thought it would be `export default Chat` or something... no love